Husky a300 AMP and Observer

rhysfaultless-cpr opened the pull request:
- Added pages in the Solutions directory for both these robots. These pages are intended to capture all the hardware documentation, without having to navigate to the A300 page.
- Set up the pages for future Component reuse in the A300 page.
- Added centred-table CSS.
- Reduced image sizes to improve page loading time.

rhysfaultless-cpr committed:
Added section for AMP Arch's cables

rhysfaultless-cpr committed:
Added section for AMP's brushes

rhysfaultless-cpr committed:
Updated content to align with existing Integration section

rhysfaultless-cpr committed:
Renamed file and Component

rhysfaultless-cpr committed:
Removed content from Integration page, as it is redundant with the User Manual.

rhysfaultless-cpr committed:
Changed link from relative to root

rhysfaultless-cpr committed:
Moved content into a Component

rhysfaultless-cpr committed:
Added figcaption to the bulkhead's image

rhysfaultless-cpr committed:
Added italics and details about the connector bulkhead.

rhysfaultless-cpr committed:
Moved the Connector Bulkhead's content from Integration to Maintenance.

rhysfaultless-cpr committed:
Moved STEP model content to a Component

rhysfaultless-cpr committed:
Moved ICD content to a Component.

rhysfaultless-cpr committed:
Moved Aluminum Extrusion content to a Component.

rhysfaultless-cpr committed:
Moved Integration Safety content to a Component.

rhysfaultless-cpr committed:
Changed from relative link to root

rhysfaultless-cpr committed:
Moved Large Integration content to a Component

rhysfaultless-cpr committed:
Removed Fusing section

rhysfaultless-cpr committed:
Deleted the Integration page, and moved the content to the User Manual.

rhysfaultless-cpr committed:
Renamed file, to remove folder structure from the Side Navbar

rhysfaultless-cpr committed:
Added TM icon

rhysfaultless-cpr committed:
Removed white background, and reduced resolution to improve page loading time.

rhysfaultless-cpr committed:
Reduced image resolution to improve page load time. Reduced from 33 MB to 1 MB.

rhysfaultless-cpr committed:
Reduced table of contents to only show <h2> tags for readability.

rhysfaultless-cpr committed:
Moved STEP model to a Component.

rhysfaultless-cpr committed:
Replaced admonition with existing Component.

rhysfaultless-cpr committed:
Moved Interfaces content out of the Risk Assessment component.

rhysfaultless-cpr committed:
Removed <br />

rhysfaultless-cpr committed:
Update links to rool, rather than relative.

rhysfaultless-cpr committed:
Added TODO comment

rhysfaultless-cpr committed:
Added indenting.

rhysfaultless-cpr committed:
Moved Integration content to the User Manual. Deleted the Integration page.

rhysfaultless-cpr committed:
Added indent.

rhysfaultless-cpr committed:
Changed the Table Of Contents to only show <h2> for readability.

rhysfaultless-cpr committed:
Renamed file to remove folder structure from the Side Navbar.

rhysfaultless-cpr committed:
Added whitespace.

rhysfaultless-cpr committed:
Updated link.

rhysfaultless-cpr committed:
Added headings.

rhysfaultless-cpr committed:
Moved initial Safety section to a Component.

rhysfaultless-cpr committed:
Added the What's Included section.

rhysfaultless-cpr committed:
Moved Overview section to a Component.

rhysfaultless-cpr committed:
Added Technical Specifications.

rhysfaultless-cpr committed:
Added section for hinging the Electronics Tray.

rhysfaultless-cpr committed:
Added images of tray removal

rhysfaultless-cpr committed:
Added subheadings

rhysfaultless-cpr committed:
Added maintenance section for removing the Battery Integration Tray.

rhysfaultless-cpr committed:
Added comments for content to create

rhysfaultless-cpr committed:
Added section for removing the Power Distribution.

rhysfaultless-cpr committed:
Added section for how to remove the batteries.

rhysfaultless-cpr committed:
Added maintenence section for the Bumpers

rhysfaultless-cpr committed:
Moved removal steps into <details> to make the page simpler to navigate.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
removed white background

rhysfaultless-cpr committed:
Renamed component

rhysfaultless-cpr committed:
Added section for removing the Rear Cover.

rhysfaultless-cpr committed:
removed shadows

rhysfaultless-cpr committed:
Changed info-admonitions to note-admonitions, to improve visibility within <details> elements.

rhysfaultless-cpr committed:
Removed motor vendor name

rhysfaultless-cpr committed:
Added section for removing the Air Grilles

rhysfaultless-cpr committed:
Removed background from image.

rhysfaultless-cpr committed:
Removed background from image.

rhysfaultless-cpr committed:
Separated A300 and AMP maintenance

rhysfaultless-cpr committed:
Added italics

rhysfaultless-cpr committed:
Renamed file and header.

rhysfaultless-cpr committed:
Changed cover and enclosure to panel.

rhysfaultless-cpr committed:
Added <details>

rhysfaultless-cpr committed:
Added note about replacement battery configuration

rhysfaultless-cpr committed:
Moved Common Replacement Items section to a reusable component. Added a section within this component for Critical Items.

rhysfaultless-cpr committed:
Added table of contents

rhysfaultless-cpr committed:
Update the TOC

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Switched image.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Changed Safety Component, to eventually reuse with Husky A300's manual.

rhysfaultless-cpr committed:
Removed <h3>

rhysfaultless-cpr committed:
Added section for intended use.

rhysfaultless-cpr committed:
Added mobility power

rhysfaultless-cpr committed:
Added section for Safety System Functionality

rhysfaultless-cpr committed:
Added <h3> into component.

rhysfaultless-cpr committed:
Moved 2 sentences down after the Disclaimer Admonition

rhysfaultless-cpr committed:
Added safety admonition

rhysfaultless-cpr committed:
Moved Danger Admonition up in the page.

rhysfaultless-cpr committed:
Added <br/>

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
changed PNG to png

rhysfaultless-cpr committed:
Typo, Upgradeed

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Added draft of Safety admonitions per Risk Assessment.

rhysfaultless-cpr committed:
Created a copy of the Safety Component, using functions for the individual admonitions. This allows me to reorder the rendered admonitions, withough moving the entire content blocks of text in the source file.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Spelling, erratic

rhysfaultless-cpr committed:
Spelling, Husky's

rhysfaultless-cpr committed:
Spelling, due to, and

rhysfaultless-cpr committed:
Spelling, include

rhysfaultless-cpr committed:
Spelling, wearing

rhysfaultless-cpr committed:
Spelling, environment's.

rhysfaultless-cpr committed:
Changed user to technician.

rhysfaultless-cpr committed:
Changed the order of rendered safety admonitions.

rhysfaultless-cpr committed:
Changed drop height from 200 mm to 150 mm.

rhysfaultless-cpr committed:
Added operating time, per testing with 80 Ah Husky AMP

rhysfaultless-cpr committed:
Added TODO comments

rhysfaultless-cpr committed:
Removed image's background.

rhysfaultless-cpr committed:
Moved list of 20+ cables into a <details>

rhysfaultless-cpr committed:
Added sections for motors, and tires

rhysfaultless-cpr committed:
Added step for removing the cable comb.

rhysfaultless-cpr committed:
Changed info admonition to a not admontion, since this is included in a <details>

rhysfaultless-cpr committed:
Removed the title from the Danger Admonition

rhysfaultless-cpr committed:
Added Preventative Maintenance section.

rhysfaultless-cpr committed:
Added Safety admonition to all of the Maintenance sections, as technicians will use links to jump around the manual.

rhysfaultless-cpr committed:
Added Battery Balancing

rhysfaultless-cpr committed:
Made procedure an Admonition.

rhysfaultless-cpr committed:
Added content about low tire pressures for rough terrain.

rhysfaultless-cpr committed:
Changed PSI to psi.

rhysfaultless-cpr committed:
Added inspection for E-stop distance

rhysfaultless-cpr committed:
Added section for replacing System Interface Assembly's PCBAs.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Moved light states to a reusable component.

rhysfaultless-cpr committed:
Added section for Emergency Stop Buttons.

rhysfaultless-cpr committed:
Added section for replacing Fans.

rhysfaultless-cpr committed:
Changed 'the' to 'The'

rhysfaultless-cpr committed:
Removed comment

rhysfaultless-cpr committed:
Added section for removing the status lights

rhysfaultless-cpr committed:
Added section for the Network Switch.

rhysfaultless-cpr committed:
Added section for removing the computer.

rhysfaultless-cpr committed:
Added section for lifting sling.

rhysfaultless-cpr committed:
Rotated image 90°.

rhysfaultless-cpr committed:
Changed System to Technical.

rhysfaultless-cpr committed:
Added section for lifting by hand.

rhysfaultless-cpr committed:
Added system architecture.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Added section related to collisions.

rhysfaultless-cpr committed:
Added footnotes

rhysfaultless-cpr committed:
Added Technical Specifications for the base A300. This component will be added to the User Manual's page later.

rhysfaultless-cpr committed:
Added diagrams.

rhysfaultless-cpr committed:
Added component for System Limits

rhysfaultless-cpr committed:
Updated to the system limits.

rhysfaultless-cpr committed:
Changed Wireless Charging images.

rhysfaultless-cpr committed:
Moved components to subdirectory, husky_a300

rhysfaultless-cpr committed:
Added dimensional drawings

rhysfaultless-cpr committed:
Added comment about the Luxonis functionalty at low temperature, per CPE87-2674.

rhysfaultless-cpr committed:
Added <br />

rhysfaultless-cpr committed:
Added IP addresses.

rhysfaultless-cpr committed:
Added hero image of the motor

rhysfaultless-cpr committed:
Removed image background

rhysfaultless-cpr committed:
Removed image background.

rhysfaultless-cpr committed:
Updated image

rhysfaultless-cpr committed:
Removed image background.

rhysfaultless-cpr committed:
Added Hero shot of the air grille

rhysfaultless-cpr committed:
Added images of the side panel and connector bulkhead.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Added image of the computer.

rhysfaultless-cpr committed:
Added image of the M.2 cards.

rhysfaultless-cpr committed:
Added images of the emergency stop buttons.

rhysfaultless-cpr committed:
Added image of the fans.

rhysfaultless-cpr committed:
Added images of the network switch.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Added image of the Rear Cover.

rhysfaultless-cpr committed:
Rotated image 0.5°.

rhysfaultless-cpr committed:
Added section about base_link

rhysfaultless-cpr committed:
Added Orientation Reference.

rhysfaultless-cpr committed:
Added centre of mass data.

rhysfaultless-cpr committed:
Updated the Stability Limits to mention the COM in the technical specifications table.

rhysfaultless-cpr committed:
Added TOC

rhysfaultless-cpr committed:
Added line to About.

rhysfaultless-cpr committed:
Moved the section Other Documentation To Review into a reusable component.

rhysfaultless-cpr committed:
Moved section for What's Included to a reusable component.

rhysfaultless-cpr committed:
Type of Component call.

rhysfaultless-cpr committed:
Added components from AMP.

rhysfaultless-cpr committed:
Updated Intended Use, using props to render lines specific to A300, AMP, or Observer.

rhysfaultless-cpr committed:
Added TODO comments.

rhysfaultless-cpr committed:
Updated the Technical Specifications to be a common component, for A300, AMP, and Observer.

rhysfaultless-cpr committed:
Added System Architecture

rhysfaultless-cpr committed:
Combined IP Address content into a single reusable component for A300, AMP, and Observer.

rhysfaultless-cpr committed:
Made Emergency Stop button component more ambiguous, to reuse with AMP and Observer.

rhysfaultless-cpr committed:
Added Safety System functionality.

rhysfaultless-cpr committed:
Reduced image size.

rhysfaultless-cpr committed:
Added Getting Started.

rhysfaultless-cpr committed:
Typos

rhysfaultless-cpr committed:
Updated Component to pass robotModel props, for reuse with A300, AMP, and Observer.

rhysfaultless-cpr committed:
Changed orientation image.

rhysfaultless-cpr committed:
Updated System Limits to include Diagnostics Monitoring.

rhysfaultless-cpr committed:
Added system components image and list for A300. Updated the Component to be reusable for A300, AMP, and Observer.

rhysfaultless-cpr committed:
Added  a note that the Wi-Fi antennas are relocated on AMP.

rhysfaultless-cpr committed:
Changed Admonition syntax, to remove icon and title.

rhysfaultless-cpr committed:
Tab alignment

rhysfaultless-cpr committed:
Added details about wireless charging

rhysfaultless-cpr committed:
Added space

rhysfaultless-cpr committed:
Moved STEP model to a reusable component.

rhysfaultless-cpr committed:
Updated STEP component to be reused for A300, AMP, and Observer.

rhysfaultless-cpr committed:
Moved PACS to a reusable component.

rhysfaultless-cpr committed:
Added commented Component for PACS. I do not intend to render this component, but want it in the page structure for reuse with A300.

rhysfaultless-cpr committed:
Moved Cable Passthrough to a reusable component.

rhysfaultless-cpr committed:
Moved Internal Integrations to a reusable component. Added the component to AMP.

rhysfaultless-cpr committed:
Moved Data Connections to a reusable Component, added this to AMP.

rhysfaultless-cpr committed:
Added AMP to the header title.

rhysfaultless-cpr committed:
Added base_link to the Husky A300 integration page.

rhysfaultless-cpr committed:
Added an admonition mentioning that the STEP Model's origin is aligned with base_link.

rhysfaultless-cpr committed:
Added AMP to the header title.

rhysfaultless-cpr committed:
Moved System Interface Connectors to a reusable component, and added this to AMP.

rhysfaultless-cpr committed:
Moved Power Connections to a reusable component, and added this to AMP.

rhysfaultless-cpr committed:
Moved Fans to a reusable component, and added this to AMP.

rhysfaultless-cpr committed:
Moved Wireless E-stop to a reusable component.

rhysfaultless-cpr committed:
Commented out Wireless E-stop. We do not want to expose this as a remote integration or custom wiring task for technicians.

rhysfaultless-cpr committed:
Added Wireless Emergency Stop to the Integration area, but commented out the Component.

rhysfaultless-cpr committed:
Moved CANbus to a reusable component, and then added this to AMP.

rhysfaultless-cpr committed:
Moved Receiver Installation subsection to the Wireless E-stop Component

rhysfaultless-cpr committed:
Moved Additional Estop Breakouts to a reusable component, and added this to AMP.

rhysfaultless-cpr committed:
Added AMP to the title header.

rhysfaultless-cpr committed:
Moved the System Power section to a reusable component, and then added this to AMP.

rhysfaultless-cpr committed:
Moved Debug LEDs section to a reusable component, and then added this to AMP.

rhysfaultless-cpr committed:
Hid Debug LEDs from the A300 Integration page.

rhysfaultless-cpr committed:
Updated common_software_integration, to also include the section header and hyperlinks. React Props are used to render the forrect <h> size.

rhysfaultless-cpr committed:
Added Software Integration section.

rhysfaultless-cpr committed:
Removed Integration Notes about Husky A300, as all the original components are now included on the AMP and Observer pages.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Added comments for Tutorial and Troubleshooting sections.

rhysfaultless-cpr committed:
Removed comment.

rhysfaultless-cpr committed:
Added section for Operating The Robot

rhysfaultless-cpr committed:
Moved support Component

rhysfaultless-cpr committed:
removed comment

rhysfaultless-cpr committed:
Copied structure and content from AMP page.

rhysfaultless-cpr committed:
Added missing import

rhysfaultless-cpr committed:
Removed comment, to render Component.

rhysfaultless-cpr committed:
Updated table centring CSS.

rhysfaultless-cpr committed:
Reduced image sizes, to 1280px for the long edge. This reduces file sizes, and improves page loading time.

rhysfaultless-cpr committed:
Added Observer hero image.

jhiggins-cpr commented on .vscode/settings.json:
```diff
@@ -0,0 +1,3 @@
+{
+  "idf.pythonInstallPath": "C:\\Users\\rfaultless\\.espressif\\tools\\idf-python\\3.11.2\\python.exe"
```
I assume this wasn't supposed to be included?

rhysfaultless-cpr replied:
I have removed the file and directory.

jhiggins-cpr commented on components/husky_a300/husky_a300_safety_alternate.mdx:
```diff
@@ -0,0 +1,798 @@
+<details>
+  <summary>Click To Expand, To Show All The Hazards</summary>
+  <!-- DANGER-->
+  <ComponentRiskAssessmentLine_29 />
+  <ComponentRiskAssessmentLine_14 />
+  <ComponentRiskAssessmentLine_26 />
+  <ComponentRiskAssessmentLine_27 />
+  <ComponentRiskAssessmentLine_30 />
+  <ComponentRiskAssessmentLine_9_10_11_12 />
+
+  <!-- WARNING-->
+  <ComponentRiskAssessmentLine_5_6_7_8 />
+  <ComponentRiskAssessmentLine_24 />
+  <ComponentRiskAssessmentLine_37 />
+  <ComponentRiskAssessmentLine_46 />
+  <ComponentRiskAssessmentLine_49_50 />
+  <ComponentRiskAssessmentLine_51_52 />
+  <ComponentRiskAssessmentLine_55 />
+  <ComponentRiskAssessmentLine_66 />
+
+  <!-- CAUTION-->
+  <ComponentRiskAssessmentLine_1_2 />
+  <ComponentRiskAssessmentLine_4 />
+  <ComponentRiskAssessmentLine_13 />
+  <ComponentRiskAssessmentLine_18_31 />
```
There is no urgent need to change these for AMP, but just noting that these are not in the format approved by the Rockwell safety team. We cannot merge these as they are into the base Husky A300.

rhysfaultless-cpr replied:
From a Standup discussion; we will merge this as is. A future change will update the Safety Admonitions, and update the A300 page to use the MDX Components from AMP.

jhiggins-cpr commented on components/husky_a300/husky_a300_safety_alternate.mdx:
```diff
@@ -0,0 +1,798 @@
+    <div>
+      <SafetyAdmonition>
+        <>
+          Custom attachmentes such as towed trailers and manipulators were not been considered in Clearpath Robotics's risk assessment process.
```
"attachments"

jhiggins-cpr commented on components/husky_a300/husky_a300_safety_alternate.mdx:
```diff
@@ -0,0 +1,798 @@
+          <br/>
+          <Admonition type="note">
+            The optional Upgraded Charger for Husky A300 may be used outdoors.
+            The AC power cable for this charger must be kept with a awning or cable-cover.
```
"an" awning

jhiggins-cpr commented on components/husky_a300/husky_a300_safety_alternate.mdx:
```diff
@@ -0,0 +1,798 @@
+      <SafetyAdmonition>
+        <>
+          Dry the Husky A300's Rear Charge Port Door before opening it.
+          Do not open the during heavy rain—_(more than 1 mm per hour)_—or when wind is angling the rainfall.
```
I don't think you got the italics you were looking for here.

jhiggins-cpr commented on components/husky_a300/husky_a300_safety_alternate.mdx:
```diff
@@ -0,0 +1,798 @@
+      <SafetyAdmonition>
+        <>
+          The robot may become unstable as you remove items for maintenance.
+          Place the entire bottom sufrace of the Husky's chassis on a table when removing wheels to maintain the robot's stability.
```
"surface"

jhiggins-cpr commented on components/husky_a300/husky_a300_components.mdx:
```diff
@@ -0,0 +1,125 @@
+  </center>
+ 
+  <ol>
+    <li>GNSS antennas and Positioning Unit, <em>(Fixposition Vision RTK 2)</em></li>
+    <li>Wi-Fi and Bluetooth antennas, <em>(relocated)</em></li>
+    <li>Cellular antennas, all-in-one unit</li>
+    <li>Teleoperation cameras, <em>(Luxonis Oak-D Pro W PoE)</em></li>
+    <li>Directional 3D lidar, <em>(Seyond Robin W)</em></li>
```
Just making sure that this was a conscious decision to identify specific sensors here in the public docs. I'm not opposed to it but I haven't discussed it with others either. Do you know if the specific sensors are already noted in other public docs that we publish?

rhysfaultless-cpr replied:
I included the sensor names for AMP since they are already published on the main website. Refer to the Included Sensors section:

https://clearpathrobotics.com/husky-a300-unmanned-ground-vehicle-robot/

jhiggins-cpr commented on components/husky_a300/husky_a300_emergency_stop_wireless.mdx:
```diff
@@ -0,0 +1,67 @@
+* Press the red button _(GSS)_ to exert an Emergency Stop.
+* Store the _Wireless Emergency Stop Transmitter_ with the red button depressed.
+  This is also the transmitter's off button.
+* The transmitter will automatically exert an Emergency Stop if it looses signal to the robot.
```
looses -> loses

jhiggins-cpr commented on components/husky_a300/husky_a300_amp_other_documentation_to_review.mdx:
```diff
@@ -0,0 +1,5 @@
+
+Refer to our [OutdoorNav software's documentation](/docs_outdoornav_user_manual), as this comes included with Husky AMP and Husky Observer.
+OutdoorNav allows you to send your Husky on autonomous missions using the included ROS API, or by using OutdoorNav's web user interface.
+OutdoorNav can accomplish these autonomous missions using the Husky AMP's sensors, without making infrasture updates to your worksite.
```
"infrastructure"

jhiggins-cpr commented on components/common_wireless_joystick_ps4.mdx:
```diff
@@ -0,0 +1,45 @@
+
+:::info Driving The Robot
+
+1.  Make sure your worksite is ready to drive the robot, and that the robot will not create hazards for people, animals, or your infrastrure.
```
"infrastructure"

jhiggins-cpr commented on components/husky_a300/husky_a300_charging_the_robot.mdx:
```diff
@@ -0,0 +1,138 @@
+| :------------------------------ | :-----------------------------------------------------------------------------------------------------: |  :-------------------------------------------------------------------------------------------------------: |
+| Standard Charger, North America | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_ready_north_america.png" width="300"/> | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_charging_north_america.png" width="300"/> |
+| Standard Charger, Europe        | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_ready_europe.png" width="300"/>        | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_charging_europe.png" width="300"/>        |
+| Upgraded Charger                | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_ready_fast.png" width="250"/>          | <img src="/img/robot_images/husky_a300_images/husky_a300_charger_charging_fast.png" width="250"/>          |
```
Is there a reason why we included the 1200W but not the 650W charger here? If the lights are the same for both, should we make a note of that?

rhysfaultless-cpr replied:
I did not include an image of the 650 W charger since it has the same light pattern as the 1200 W charger. I have added a footnote to mention the 650 W charger.

jhiggins-cpr commented on components/husky_a300/husky_a300_system_limits.mdx:
```diff
@@ -0,0 +1,213 @@
+#### Diagnostics Monitoring
+
+The Husky has protection software and hardware to reduce the chance of damaging components.
+For example, the motor controllers's firmware will reduce the allowable mobility power if the motors are near their thermal limit.
```
"controller's"

jhiggins-cpr commented on components/husky_a300/husky_a300_system_limits.mdx:
```diff
@@ -0,0 +1,213 @@
+:::note Emergency Stop Braking Limits Footnotes
+
+The expected steady state rolling speed after pressing an Emergency Stop button.
+The resistive shunt brake increates in braking force linearly with the motor's speed.
```
increases

jhiggins-cpr commented on components/husky_a300/husky_a300_battery_balancing_full.mdx:
```diff
@@ -0,0 +1,15 @@
+### Battery Cell Balancing _(Full)_
```
Can you review this section with JasonB? I think he had me remove it for the Inventus batteries.

rhysfaultless-cpr replied:
We reviewed in person. I have updated the file per the discussion.

jhiggins-cpr commented on components/husky_a300/husky_a300_battery_balancing_quick.mdx:
```diff
@@ -0,0 +1,35 @@
+import Admonition from "@theme/Admonition";
+
+### Battery Cell Balancing _(Quick)_  {#quick-cell-balance}
```
Can you review this section with JasonB? I think he had me remove it for the Inventus batteries.

rhysfaultless-cpr replied:
We reviewed in person. I have removed this file due to the discussion.

jhiggins-cpr commented on components/husky_a300/husky_a300_nameplate.mdx:
```diff
@@ -0,0 +1,22 @@
+      src="/img/robot_images/husky_a300_images/husky_a300_nameplate.jpg"
+      width="700"
+    />
+    <figcaption>Husky AMP, With The Sensor Arch At The Back Of The Robot</figcaption>
```
I think this is the wrong caption

jhiggins-cpr commented on components/husky_a300/husky_a300_air_grilles.mdx:
```diff
@@ -0,0 +1,47 @@
+          />
+        </figure>
+      </center>
+  4.  remove the grilles.
```
"Remove"

jhiggins-cpr commented on components/husky_a300/husky_a300_network_switch.mdx:
```diff
@@ -0,0 +1,62 @@
+  />
+</center>
+
+The network switch connects the primary computer, the rear debigging port, and any additional Ethernet devices.
```
"debugging"

jhiggins-cpr reviewed (commented):
Thanks for putting so much thought and effort into these changes. I expect that this will be very helpful for customers, integrators, and client success. Mostly minor comments throughout. A few top-level points:
1.  Can we remove A200 Observer from the main Solutions and just keep it under ROS 1 Solutions?
2. We may want to refine the AMP Getting Started section (to combine with OutdoorNav somewhat), but will cover that in a separate merge request.
3. The section number is not working as expected. Specifically, the second number does not reset when the first number increments. For example, the AMP User Manual goes from Section 1. 5 to 2.6 (rather than 2.1).
4. We should discuss the format for the safety admonitions that has been requested by the Rockwell safety team.

rhysfaultless-cpr committed:
Fixing typos per review comments.

rhysfaultless-cpr committed:
Typo, attachmentes to attachments

rhysfaultless-cpr committed:
Changed syntax from Markdown to HTML.

rhysfaultless-cpr committed:
Added footnote to mention 650 W upgraded charger.

rhysfaultless-cpr committed:
Updated image's caption.

rhysfaultless-cpr committed:
Removed A200 Observer from current Solutions directory. Updated page links to route to the Legacy pages.

rhysfaultless-cpr committed:
Simplified the sidebar labels

rhysfaultless-cpr committed:
Removed Battery Balancing Full, and updated the Quick balancing based on internal discussion for Inventus batteries.

rhysfaultless-cpr committed:
Renamed file.

rhysfaultless-cpr committed:
Changed can0 to vcan0, and added the Wireless Emergency  Stop.

rhysfaultless-cpr committed:
Removed file and directory.

rhysfaultless-cpr commented:
Per @jhiggins-cpr earlier comments:

1. I have updated the files to remove Husky A200 Observer from /docs_robots/solutions/
2. I will leave the files as is for now.
3. This was related to a CSS minification issue. I have updated AWS Amplify's build settings, and explained the issue in my personal wiki notes.
4. I will leave the files as is for now.
